README.md

```diff
@@ -1,17 +1,86 @@
+
+#### Storage
+
+| **Term**        | **Suggested Usage**                                                                                                                          |
+|-----------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| Storage Service | Write as the "Storage Service" (as a proper noun).                                                                                            |
+| Bucket          | Write as "a bucket" or "buckets", not a proper noun.                                                                                          |
+| File            | Write as "a file" or "files", not a proper noun.                                                                                              |
+
+#### Functions
+| **Term**              | **Suggested Usage**                                                                                                                                                        |
+|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Functions Service     | Write as the "Functions Service"(as a proper noun). Not "Function Service" or "Cloud Functions".                                                                            |
+| Function (individual) | When we mention a specific function and not the service as a whole, use "a function" or "functions". Not "a cloud function" or "a Function".                                |
+| Create                | When we "create" a function, it refers to the process of using "Add Function" in console or using `functions.create()` method. No code is uploaded.                         |
+| Deploy/Deployment     | When we say we "deploy" a function or create a new deployment, this is when we upload code through the Create Deployment button or endpoint.                                |
+| Activate Deployment   | When we enable a particular version of a function, we say we "activate" the deployment.                                                                                     |
+| Runtimes              | When we refer to a runtime like `node.js 15.5`, call it a "Node.js runtime" or "function runtime". Avoid similar terms like "runtime environment" or "function environment" |
+| Execution            | When a function is run, triggered by an event, or triggerd by a CRON job, an "execution" is created. In documentation, refer to "running" a function as "creating an execution".|
+| Variables            | When a function is run, a set of variables are passed in through the `req` object. Some are generated by the runtime, some are defined under the settigs of a function. These are referred to as "variables" and not "environment variables".|
```

Comment:
typo and rather than:

```suggestion
| Variables            | When a function is run, a set of variables are passed in through the `req` object. Some are generated by the runtime, some are defined under the settings of a function. These are referred to as "variables" rather than "environment variables".|
```

It's a little confusing because they end up in the `req.env` variable.